test/psych/test_psych.rb

```diff
@@ -57,7 +57,7 @@ def test_non_existing_class_on_deserialize
   end
 
   def test_dump_stream
-    things = [22, "foo \n", {}]
+    things = [22, "foo", {}]
```

Comment:
Probably due to some bug in @Iazel’s implementation that I’ve fixed afterwards — this change is no longer needed.

I’ll send updated PR rebased against current master…